src/components/PageElements/Elements/ReviewApplicantResponse.tsx

```diff
@@ -156,14 +173,16 @@ const ReviewApplicantResponse: React.FC<ReviewApplicantResponseProps> = ({
                 applicationResponse={applicationResponse}
                 summaryViewProps={summaryViewProps}
               >
-                {!decisionExists && (
-                  <ReviewElementTrigger
-                    title={triggerTitle} // Review or Re-review
-                    onClick={() => setIsActiveEdit(true)}
-                  />
-                )}
+                {!decisionExists &&
+                  (reviewType === 'OptionallyReviewable' ? (
+                    <AddIcon onClick={() => setIsActiveEdit(true)} />
```

Comment:
I thought this "+" icon would be a suitable alternative for the "optionally" reviewable items. It needs to be significantly different, as when you click "Approve All", these ones won't be approved, so we want to avoid confusion as to why not.

Open to other suggestions though?